common/main/mai.xml

```diff
@@ -4118,6 +4120,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			<metazone type="Samoa">
 				<long>
 					<generic>सामोआ समय</generic>
+					<standard>सामोआ मानक समय</standard>
+					<daylight>सामोआ समर समय</daylight>
```

Comment:
Isn't daylight: "डेलाइट समय"

Reply:
समर is the form of "daylight" that is used in e.g. Sakhalin, Philippines, etc. They have a different form for the word for "time" however. I kept the word for time that was already used here.

Reply:
We certainly have the ability to further adjust these once we have resolved the errors so that we can merge the vxml branch to main.